config.json

```
@@ -60,6 +60,16 @@
         "arrays"
       ]
     },
+    {
+      "slug": "armstrong-numbers",
+      "uuid": "77c5c7e6-265a-4f1a-9bc4-851f8f825420",
+      "core": false,
+      "unlocked_by": "isogram",
```

@guygastineau I didn't realize before that Isogram has moved since the last time we discussed adding this exercise. Let's make it    
```suggestion
      "unlocked_by": "series",
```
Sorry that I missed this.

---

Note to self: without the lambda, it could be a level 1 side exercise. Then, it may need a more step by step help in the readme. 

(This doesn't need to be addressed in this PR, unless you'd prefer to do so, Guy. )

---

Great, I figured there might need to be some changes to it in the `config.json`.
I'm on it 😃

---

I mean, I will change it to unlock from `series` I don't especially feel like writing step-by-step instructions for closures in a lvl 1 exercise readme extension.

---

changed